data/db.js

```diff
@@ -0,0 +1,31 @@
+const knex =require('knex');
+const dbConfig = require('../knexfile.js');
+const db = knex(dbConfig.development);
+
+module.exports = {
+    getDishes,
+    addDish,
+    getDishById,
+    getRecipes,
+    addRecipe
+}
+
+function getDishes(){
+    return db('dish')
+}
+
+function addDish(dish){
+    return db('dish')
+    .insert(dish)
+}
+function getDishById(id){
+    return db('dish').where({'dish.id': id}) ///How to include related dishes
```

Comment:
Depending on what you mean here, you could have an extra table with types of 'relations' that dishes could have, and then another table representing those relationships (since it'd be many to many).